README.md

```diff
@@ -1,15 +1,25 @@
+[![Review Assignment Due Date](https://classroom.github.com/assets/deadline-readme-button-24ddc0f5d75046c5622901739e7c5dd533143b0c8e959d652212380cedb1ea36.svg)](https://classroom.github.com/a/FgMJElkj)
 # Theory vs. Practice
 
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
+    1. Asymptotic analysis doesn't take into account constant factors. So while the constant factors do take time to run or memory to store, they aren't analysed. 
+    2. Asymptotic analysis looks at the growth rate as the input increases, but for small input sizes one algorithm might be better than another even though it's asymptotic analysis suggests otherwise. Because of recussive call overhead or other factors.
```

Comment:
Why would that be the case? Sounds like that would be just a constant factor as in your previous reason?

Reply:
Still relevant.